components/raftstore-v2/src/operation/command/write/mod.rs

```diff
@@ -90,7 +88,7 @@ impl<EK: KvEngine, ER: RaftEngine> Peer<EK, ER> {
         if !self.serving() {
             return;
         }
-        let bin = SimpleWriteReqEncoder::new(
+        let bin = SimpleWriteReqEncoder::<CmdResChannel>::new(
```

Comment:
You can reexport `pub type SimpleWriteReqEncoder = SimpleWriteReqEncoder<CmdResChannel>;`